### PR TITLE
add option to disable automatic reboots

### PR DESCRIPTION
**Pull request by CaptainTyphoo**

### CODE REVIEW

File: dnf-automatic-restart [hunk 1]
```diff
@@ -75,11 +80,15 @@ trigger_reboot() {
...
 numeric='^[0-9]+$'
+disable_reboot=false
 no_reboot_from=
 no_reboot_to=
 reboot_at=
-while getopts ":hn:r:" opt; do
+while getopts ":dhn:r:" opt; do
   case "$opt" in
+      d)
+       disable_reboot=true
```
Comment (agross) [inline dnf-automatic-restart]: Please add a space before `disable_reboot`.

File: README.md [hunk 2]
```diff
@@ -101,6 +102,12 @@ ExecStartPost=/usr/local/sbin/dnf-automatic-restart -n 8-22
 ExecStartPost=/usr/local/sbin/dnf-automatic-restart -n 8-22 -r 0
 ```
 
+```ini
+[Service]
+# No automatic reboots, only services are restarted
```
Comment (agross) [inline README.md]: Please add a `.` after the comment.

**Review by agross — CHANGES_REQUESTED** [review]
Looking good so far! Please make the changes as requested and I'll merge.

**Comment by agross** [comment]
Excellent, thanks for the update. For cleanliness, now please squash both commits into one. (You could have amended, no problems with that.)

```
git rebase -i d06b77d1a~
```

In the editor, change the second `pick` line (with your latest commit) to `fixup`, save and close the editor. Then `git push -f`. Not sure about your git experience, hence the explanation!

Thanks!

**Comment by CaptainTyphoo** [comment]
The explanation was exactly what I needed. Thank you!

**Comment by agross** [comment]
:+1:

**Pull request merged**